OptimoveSDK/optimove-sdk/src/main/java/com/optimove/sdk/optimove_sdk/kumulos/KumulosInApp.java

```diff
@@ -0,0 +1,254 @@
+package com.optimove.sdk.optimove_sdk.kumulos;
+
+import android.app.Application;
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.List;
+
+public class KumulosInApp {
+    static InAppDeepLinkHandlerInterface inAppDeepLinkHandler = null;
+    static Application application;
+    static InAppMessagePresenter presenter;
+
+    public enum InboxMessagePresentationResult {
+        FAILED,
+        FAILED_EXPIRED,
+        PRESENTED
+    }
+
+    public interface InAppInboxUpdatedHandler extends Runnable {
+        void run();
+    }
+
+    private static InAppInboxUpdatedHandler inboxUpdatedHandler;
+
+    public interface InAppInboxSummaryHandler {
+        void run(@Nullable InAppInboxSummary summary);
+    }
+
+    //==============================================================================================
+    //-- Public APIs
+
+    /**
+     * Returns up to 50 non-expired in-app messages stored in inbox
+     *
+     * @param context
+     */
+    public static List<InAppInboxItem> getInboxItems(@NonNull Context context) {
```

Comment:
public methods from here are not exposed (except directly doing KumulosInApp.*). Do we keep the class public, rename KumulosInApp -> OptimobileInApp or create a wrapper like for Kumulos.* ? Mb no point to have a wrapper

Reply:
No point having a wrapper, this is the Optimove SDK now, so rename and / or remove any Kumulos refs, it might even be better to just call it InApp as the package name if possible?

Reply:
We might want to consider having a wrapper depending which package the public API should sit in? Would be a shame to need a breaking change just if we want to shuffle the package around later.. Not sure.